spec.html

```diff
@@ -27137,31 +27143,257 @@
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  1
+                </td>
+                <td>
+                  `"Feb"`
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  2
+                </td>
+                <td>
+                  `"Mar"`
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  3
+                </td>
+                <td>
+                  `"Apr"`
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  4
+                </td>
+                <td>
+                  `"May"`
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  5
+                </td>
+                <td>
+                  `"Jun"`
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  6
+                </td>
+                <td>
+                  `"Jul"`
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  7
+                </td>
+                <td>
+                  `"Aug"`
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  8
+                </td>
+                <td>
+                  `"Sep"`
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  9
+                </td>
+                <td>
+                  `"Oct"`
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  10
+                </td>
+                <td>
+                  `"Nov"`
+                </td>
+              </tr>
+              <tr>
+                <td>
+                  11
+                </td>
+                <td>
+                  `"Dec"`
+                </td>
+              </tr>
+              </tbody>
+            </table>
+          </emu-table>
+        </emu-clause>
+
+        <emu-clause id="sec-timezoneestring" aoid="TimeZoneString">
+          <h1>Runtime Semantics: TimeZoneString( _tv_, )</h1>
+          <p>The following steps are performed:</p>
+          <emu-alg>
+            1. Assert: Type(_tv_) is Number.
+            1. Assert: _tv_ is not *NaN*.
+            1. Let _offset_ be (LocalTZA + DaylightSavingTA(_tv_)) / msPerSecond.
+            1. Assert: _offset_ is an integer, and -100000 &lt; _offset_ &lt; 100000.
+            1. If _offset_ &ge; 0, let _offsetSign_ be `"+"`; otherwise, let _offsetSign_ be `"-"`.
+            1. Let _offsetString_ be abs(_offset_) formatted as a four-digit number, padded to the left with zeros if necessary.
+            1. Let _tzName_ be an implementation-defined string, either `""` or a string of the form `" ("` _name_ `")"` where _name_ is an implementation-defined timezone name.
```

Comment:
Would it make sense to give a recommendation while leaving it still up to an implementation about _tzName_?  
If it does, it can be recommended to use the canonicalized timezone in Ecma402. Perhaps, cross-referencing is not  a good idea?   
Just my 2 cents.